src/ID3Frames.js

```diff
@@ -395,9 +399,10 @@ module.exports.WXXX = {
 
 module.exports.ETCO = {
     create: (data) => {
-        const builder = new ID3FrameBuilder("ETCO").appendStaticNumber(data.timeStampFormat, 1)
-        data.keyEvents = data.keyEvents || []
-        data.keyEvents.forEach((keyEvent) => {
+        const builder = new ID3FrameBuilder("ETCO")
+            .appendStaticNumber(data.timeStampFormat, 1)
+        const keyEvents = data.keyEvents || []
```

Comment:
I am not sure why you wanted to make `keyEvents` optional, I don't see the benefit, we surely want to write events. I would not make it optional.